.circleci/conditional_config.yml

```diff
@@ -397,12 +402,16 @@ jobs:
           root: .
           paths:
             - src/api/coverage_results
-      - store_artifacts:
-          path: ./src/api/log
-          destination: feature
-      - <<: *store_capybara_artifacts
       - store_test_results:
           path: /home/frontend/feature
+      - run: *compress_log_artifacts
+      - store_artifacts:
+          path: ./src/api/log
+          destination: log
+      - run: *compress_capybara_artifacts
+      - store_artifacts:
```

Comment:
This step is always happening after `compress_capybara_artifacts`. Could we also extract this like you did with `compress_capybara_artifacts` and re-use this code wherever needed?